package-lock.json.orig

```diff
@@ -0,0 +1,5881 @@
+{
```

Comment:
I think vim created this extra files during merge.

Reply:
it's from my git configuration